ReactiveExtensionsTests/TestHelpers/Event-Extensions.swift

```diff
@@ -1,29 +1,24 @@
-import ReactiveCocoa
+import ReactiveSwift
 
 internal extension Event {
-  internal var isNext: Bool {
-    if case .Next = self {
-      return true
-    }
-    return false
-  }
 
-  internal var isCompleted: Bool {
-    if case .Completed = self {
+  @available(*, deprecated, message: "Rename this to `isValue`.")
+  internal var isNext: Bool {
```

Comment:
we should really rename this to `isValue` to align with the new case names in `Event`, but I don't wanna introduce any source breaking changes with the Swift 3 upgrade. so, just gonna deprecate for now and it'll be a reminder for us to fix later